docs/api/portable-stories/portable-stories-vitest.mdx

```diff
@@ -13,20 +13,24 @@ draft: true
   {/* End non-supported renderers */}
 </If>
 
+<If renderer="svelte">
+  (⚠️ **Experimental**)
```

Comment:
@shilman @vanessayuenn FYI we are adding this to svelte specifically, as the API might need some extra work